### PR TITLE
Prevent links from opening in new tabs

### LINKED ISSUE #458: Don't open links in new tab.
Please open links in the same tab, e.g. GCI pages.

### PR DESCRIPTION
**Pull request by PulkitMishra**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://pulkitmishra.github.io/gci18.fossasia.org/)

- Fixes #458 

## Description / Changes

@mariobehling  opened issue #458 so that links don't open in the same tab. Necessary changes have been made and links now open in the same frame.

### Before:
N.A.
### After:
N.A.

- - - - - - - - - - - -

### CODE REVIEW

**Comment by PulkitMishra** [comment]
@poush thanks for the suggestion. Noted.

**Comment by PulkitMishra** [comment]
@fragm3 Can you please elaborate on what exactly you are looking for?
If I'm able to understand correctly do you mean to say that the 'Projects' page should open in the same frame and everything else should remain same as before?
Would be grateful if you clarify.

**Comment by fragm3** [comment]
@PulkitMishra yes only  'Projects' page should open in the same frame and everything else should remain same as before

File: students_projects.html [hunk 1]
```diff
@@ -40,7 +40,7 @@ <h1> <strong>Student Projects</strong> </h1>
                                <hr class="student-project-hr">
                                <div class="student-project-footer-container">
                                   <img src="https://github.com/{{project.github}}.png" class="img-circle student-project-creator-img">
-                                  <a href="https://github.com//{{project.github}}" target="_self" class="pull-right">{{project.creator}}</a>
+                                  <a href="https://github.com//{{project.github}}" target="_blank" class="pull-right">{{project.creator}}</a>
```
Comment (sk9331657) [inline students_projects.html]: @PulkitMishra  This should be _self, as the project should open in the same frame.

**Comment by abishekvashok** [comment]
@frag3 he mentioned gci pages clearly there, I guess student project is not only the required ones.

File: students_projects.html (hunk 1 above)
Comment (PulkitMishra) [reply students_projects.html]: @sk9331657 done. Thanks for pointing that I missed it out.

**Comment by PulkitMishra** [comment]
@abishekvashok that's what I thought. @mariobehling mentioned that he wants it like GCI pages. Hence initially I changed all occurrences of the target attribute. 
However, @fragm3 later suggested that I make it the way he describes it (which also makes sense tbh) so I made changes accordingly.

**Comment by fragm3** [comment]
@PulkitMishra @abishekvashok isn't it how it's supposed to work? Opening external pages in new tab, that's standard behaviour. Opening external websites would lead to exiting from gci website

**Comment by jogendra** [comment]
@PulkitMishra Please submit the task

**Comment by abishekvashok** [comment]
@jogendra the pr contained typos :/ 
The issue wasn't fully solved, reopening the issue :)

**Pull request merged**